warehouse/models/staging/transit_database/stg_transit_database__county_geography.sql

```diff
@@ -18,8 +18,12 @@ stg_transit_database__county_geography AS (
         rtpa,
         mpo,
         place_geography,
+        unnested_organizations AS organizations,
+        unnested_services AS services,
         dt
     FROM once_daily_county_geography
+    LEFT JOIN UNNEST(once_daily_county_geography.organizations) AS unnested_organizations
```

Comment:
I actually think we probably want to remove these UNNESTS -- we only `UNNEST` in the staging table if there's only one element per array in this table, otherwise just leave it as an array and handle later (see for example how we don't unnest `mobility_services_managed` in the `organizations` staging table, just leave it as an array) -- I would just add `organizations` and `services` to the column selection list, and then add them as array columns in the intermediate table when you make the dimension

Reply:
Looked at this again, I still don't think we want to unnest this here -- on the county geography side this is a 1:many relationship so this makes many rows with the same source record ID